config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_alloydbinstances.alloydb.cnrm.cloud.google.com.yaml

```diff
@@ -204,6 +204,18 @@ spec:
                       external networks set on the same instance.
                     type: boolean
                 type: object
+              pscInstanceConfig:
+                description: Configuration for Private Service Connect (PSC) for the
+                  instance.
+                properties:
+                  allowedConsumerProjects:
```

Comment:
Could you turn this field into a reference?
https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/1907/files#r1621568739